app-android/app/src/main/java/de/tutao/tutanota/AndroidNativeCryptoFacade.kt

```diff
@@ -186,15 +201,18 @@ constructor(
 		val outputFile = File(tempDir.encrypt, getFileInfo(context, parsedFileUri).name)
 		val inputStream = CountingInputStream(context.contentResolver.openInputStream(parsedFileUri))
 		val out: OutputStream = FileOutputStream(outputFile)
-		aesEncrypt(key.data, inputStream, out, iv.data, useMac = true)
+		aesEncrypt(key.data, inputStream, out, iv.data, useMac = true, usePadding = true)
 		return EncryptedFileInfo(outputFile.toUri(), inputStream.byteCount.toInt())
 	}
 
 	@Throws(CryptoError::class, IOException::class)
-	fun aesEncrypt(key: ByteArray, input: InputStream, out: OutputStream, iv: ByteArray, useMac: Boolean) {
+	fun aesEncrypt(key: ByteArray, input: InputStream, out: OutputStream, iv: ByteArray, useMac: Boolean = true, usePadding: Boolean = true) {
 		var encrypted: InputStream? = null
 		try {
-			val cipher = Cipher.getInstance(AES_MODE_PADDING)
+			val cipher = when (usePadding) {
```

Comment:
this could be an `if` (`if`s are also expressions)

Reply:
Cool! Fixed.